lambify/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy
+on: workflow_dispatch
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Environment
+        run: |
+          echo "UID=$(id -u)" >> $GITHUB_ENV
+          echo "GID=$(id -g)" >> $GITHUB_ENV
```

Comment:
You may be wondering why this is here. Caching brought up an important issue (sorta) with Linux and Docker. Docker for Mac will automatically keep the user/group for shared volumes in sync. When doing caching for GitHub Actions, I ran into a lot of files and folders that could not be cached due to them being owned by root. The GitHub runner does not have `UID` and `GID` environment variables set. So we help it along a bit. Each of the actions will have this and I will point out how this connects with the changes in `docker-compose.yml` further below. If you are not on Linux, things still work as normal.

Reply:
WHOOPS! Forgot this critical change with the PR. https://github.com/customink/lamby-cookiecutter/commit/cf45c3b097bab2485276d1ce79a437fd091dfd1c